lib/screens/health/health_screen.dart

```diff
@@ -36,28 +35,34 @@ class _HealthScreenState extends State<HealthScreen> {
   }
 
   Widget _content(HealthScreenViewModel viewModel) {
-    if (viewModel.showProjectEmptyScreen) {
+    if (viewModel.showLoadingScreen) {
+      return Scaffold(body:
```

Comment:
Do we need the `Scaffold` widget?

Reply:
I wasn't able to position it center otherwise with loading indicator and text 🙈

Reply:
Don't think we are supposed to use Scaffold in this place, i will give it a try.

Reply:
Works without it and we also have white as background color as in other states.